app/observers/manual_observers_registry.rb

```diff
@@ -19,6 +23,14 @@ def creation
     []
   end
 
+  def withdrawal
+    [
+      publishing_api_withdrawer,
+      rummager_withdrawer,
+      panopticon_exporter,
+    ]
```

Comment:
Should we be creating a `PublicationLog` record when withdrawing? It seems like an important change which would be useful to log.